app/javascript/components/batch_update/components/App.js

```diff
@@ -301,8 +305,22 @@ class App extends React.Component {
       })
   }
 
+  onAddRestrictedArtworkIDs(artworkIDs) {
+    this.setState({
+      restrictedArtworkIDs: artworkIDs
+        .split(' ')
```

Comment:
Do we feel confident that a single space is good for splitting (as opposed to a regex like `/\s+/`?)

Reply:
Was thinking the same thing! Should be updated in the diff now :+1: